web/thesauruses/_meta/trees.json

```diff
@@ -0,0 +1,94 @@
+{
+  "meta": {
+    "structure": "trees",
+    "structure_name": "Trees"
+  },
+  "categories": {
+    "Creating Basic Tree Objects":{
+        "structure_of_node": "What data structure do you create the tree node with?",
+        "information_inside_node": "Information stored inside of a node",
+        "create_node_class": "Creating the node class or struct",
+        "create_a_node_object": "Creating an object of the node class or struct",
+        "create_tree_class": "Creating the tree class",
+        "create_a_tree_object": "Creating an object of the tree class",
+        "create_node_for_tst": "Creating a node for a ternary search tree (TST)",
+        "create_n_ary_trees": "Creating N-ary trees",
+        "create_btree_node": "Creating a node for a B-tree",
+        "create_btree": "Creating a B-tree",
+        "create_b+tree_node": "Creating a node for a B+ tree",
+        "create_b+tree": "Creating a B+ tree"
```

Comment:
Similarly there might be multiple kinds of trees in the language (like [Java seems to have a billion of them](https://docs.oracle.com/en/java/javase/25/docs/api/jdk.compiler/com/sun/source/tree/Tree.html)). Maybe these could be `class_name_for_a_...._tree` type fields.